package.json

```diff
@@ -10,17 +10,26 @@
     "@theforeman/vendor": ">= 6.0.0"
   },
   "dependencies": {
-    "react-json-tree": "^0.11.0"
+    "prop-types": "^15.7.2",
+    "react": "^17.0.1",
+    "react-json-tree": "^0.11.2"
   },
   "devDependencies": {
     "@babel/core": "^7.7.0",
+    "@patternfly/react-core": "^4.84.4",
+    "@patternfly/react-table": "^4.19.45",
     "@theforeman/builder": "^6.0.0",
     "@theforeman/eslint-plugin-foreman": "6.0.0",
     "@theforeman/test": "^6.0.0",
+    "@theforeman/vendor": "^8.1.2",
```

Comment:
Foreman is currently on ^6.0.0, it should be ok but I am not sure if plugins should be ahead...